tools/docker/builder/openwrt/Dockerfile

```diff
@@ -46,68 +62,40 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
 
 WORKDIR /home/openwrt/openwrt
 
-COPY --chown=openwrt:openwrt configs/sdk_${SDK_CONFIG}_config /home/openwrt/sdk_config
+COPY --chown=openwrt:openwrt configs/base_${BASE_CONFIG}_config /home/openwrt/base_config
```

Comment:
HOME is not set as an argument to the Dockerfile, so not accessible here.
We could add it, but since the actual username or path used inside the container don't really matter, I'm not sure it's worth it.

Reply:
Shell commands executing via `RUN` may use not only env vars defined via `ENV` or `ARG`, some basic env vars like `HOME` are also present.
You may try to run next to see available env vars:
```
FROM ubuntu:18.04
RUN useradd -ms /bin/bash openwrt
USER openwrt
WORKDIR $HOME
RUN env
```

Reply:
It's true for shell commands executed via RUN (so true for my other comment), but neither for COPY nor for WORKDIR (or it has changed in recent docker versions - does what you pasted work for you? With which docker version?).

So yes we can use it for the `cp` command below, but we still have to hard-code it here and when using WORKDIR, so there is not much value in switching to $HOME in some places if it doesn't work in others.

What we could do if we really wanted to use another username, is to create a home directory that doesn't have the user name in it's path, but again I can't come up with a use-case where you would need to replicate the user name and home path inside the container.

Reply:
I withdraw my proposal, i didn't know that shell env vars doesn't work for COPY, WORKDIR (and maybe to some another Dockerfile specific commands too). So if it's impossible to avoid code duplication then my suggestion doesn 't make sense.